### DISCUSSION TITLE
Send to txt2img ControlNet has no effect?

### DISCUSSION THREAD

**Discussion by onewoodmot** [AlUlkesh/stable-diffusion-webui-images-browser#120] [Q&A]
apology if this has been asked before which I have not been able to find any, but after recently install, i noticed the Send to txt2img ControlNet Button does nothing? Is that something newly broken or never worked?
Thanks!

**Comment by AlUlkesh** [comment]
Yes, I'm still working on making it function again. For updates see here: https://github.com/AlUlkesh/stable-diffusion-webui-images-browser/issues/100#issuecomment-1492241152

**Comment by AlUlkesh** [comment]
Should be working again: https://github.com/AlUlkesh/stable-diffusion-webui-images-browser/commit/42293adb8c09b2875042be6c183b5fee917f0398
